redis_test.go

```diff
@@ -35,15 +36,15 @@ func testRedis(t *testing.T) *redis.Client {
 
 func Test_redisClient(t *testing.T) {
 	client := testRedis(t)
-	encodings := map[string]Encoding{
-		"gob":          GobEncoding,
-		"json":         JsonEncoding,
-		"literal+gob":  NewLiteralEncoding(GobEncoding),
-		"literal+json": NewLiteralEncoding(JsonEncoding),
+	encodings := map[string]encoding.ValueEncoding{
+		"gob":          gobEncoding,
+		"json":         encoding.JsonEncoding,
```

Comment:
`JsonEncoding` is a golint error:
```
json_encoding.go:8:5: var `JsonEncoding` should be `JSONEncoding` (golint)
var JsonEncoding = NewJsonEncoding()
    ^
```

I think golint should run in CI on go-encoding before using it here

Reply:
I'm adding golangci in ci and adding the golint linter in golangci 👍